src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -173,6 +173,11 @@ public function getConfigTreeBuilder()
                                     ->prototype('scalar')->end()
                                     ->info('URLs of the Varnish servers to purge using cache tags when a resource is updated.')
                                 ->end()
+                                ->arrayNode('request_options')
+                                    ->defaultValue([])
+                                    ->prototype('scalar')->end()
+                                    ->info('To pass options to the client charged with the request.')
```

Comment:
Some options aren't scalar but arrays, for instance: http://docs.guzzlephp.org/en/stable/request-options.html#allow-redirects

Can you update the config accordingly and add a test for this case please? https://gist.github.com/gergelypolonkai/30440e25f7a447730064 may help.

Reply:
Ok. In your example, the attribute list is hard coding (transition and time). We using Guzzle client, tomorrow it may be another client with other attributes. We should therefore pass through this code again ?

If so, I will also declare options of scalar type in order to be consistent.

Reply:
You use Guzzle client, tomorrow it may be another client with other attributes.
I thinks that the "variableNode" is perfect for this case. What do you thin

Reply:
Yes, variable is perfect.